aiida_quantumespresso/parsers/parse_xml/pw/schemas_cp/README.md

```diff
@@ -0,0 +1,6 @@
+# Note on CP schema
```

Comment:
With the new `xmlschema` library, do we still need two schemas folders? If so, let's not use the folder structure `parse_xml/pw/schemas_cp/README.md` (i.e. schemas-cp inside pw) but use `parse_xml/cp/schemas/`

Reply:
see 5fbc3ef5 major refactoring